x/marketplace/README.md

```diff
@@ -199,6 +223,34 @@ cudos-noded tx marketplace unverify-collection <collection-id> --keyring-backend
 cudos-noded tx marketplace transfer-admin-permission <new-admin> --keyring-backend=<keyring> --chain-id=cudos-local-network --gas=auto --gas-adjustment=1.3 --gas-prices=5000000000000acudos --from=<from-key>
 ```
 
+### `update-royalties`
```

Comment:
this is missing in the main section of the readme ( where everything is linked )